DNN Platform/Library/Common/Globals.cs

```diff
@@ -594,63 +660,86 @@ public static void Redirect(string url, bool endResponse)
         /// <param name="result">Data reader that contains all field values serialized.</param>
-        /// <param name="FixedColumns">List of fixed columns, delimited by commas. Columns must be contained in DataReader.</param>
-        /// <param name="VariableColumns">List of variable columns, delimited by commas. Columns must be contained in DataReader.</param>
-        /// <param name="KeyColumn">Name of the column, that contains the row ID. Column must be contained in DataReader.</param>
-        /// <param name="FieldColumn">Name of the column, that contains the field name. Column must be contained in DataReader.</param>
-        /// <param name="FieldTypeColumn">Name of the column, that contains the field type name. Column must be contained in DataReader.</param>
-        /// <param name="StringValueColumn">Name of the column, that contains the field value, if stored as string. Column must be contained in DataReader.</param>
-        /// <param name="NumericValueColumn">Name of the column, that contains the field value, if stored as number. Column must be contained in DataReader.</param>
-        /// <param name="Culture">culture of the field values in data reader's string value column.</param>
+        /// <param name="fixedColumns">List of fixed columns, delimited by commas. Columns must be contained in DataReader.</param>
+        /// <param name="variableColumns">List of variable columns, delimited by commas. Columns must be contained in DataReader.</param>
+        /// <param name="keyColumn">Name of the column, that contains the row ID. Column must be contained in DataReader.</param>
+        /// <param name="fieldColumn">Name of the column, that contains the field name. Column must be contained in DataReader.</param>
+        /// <param name="fieldTypeColumn">Name of the column, that contains the field type name. Column must be contained in DataReader.</param>
+        /// <param name="stringValueColumn">Name of the column, that contains the field value, if stored as string. Column must be contained in DataReader.</param>
+        /// <param name="numericValueColumn">Name of the column, that contains the field value, if stored as number. Column must be contained in DataReader.</param>
+        /// <param name="culture">culture of the field values in data reader's string value column.</param>
         /// <returns>The generated DataSet.</returns>
-        /// -----------------------------------------------------------------------------
-        public static DataSet BuildCrossTabDataSet(string DataSetName, IDataReader result, string FixedColumns, string VariableColumns, string KeyColumn, string FieldColumn, string FieldTypeColumn,
-                                                   string StringValueColumn, string NumericValueColumn, CultureInfo Culture)
+        [Obsolete("Deprecated in v9.8.1, scheduled for removal in v10")]
+        public static DataSet BuildCrossTabDataSet(
+            string dataSetName,

```

Comment:
Renaming parameters is a [source breaking change](https://docs.microsoft.com/en-us/dotnet/standard/library-guidance/breaking-changes#source-breaking-change) for clients using named parameters (i.e. it doesn't break at runtime, but will break on the next compile).  I think I'm okay introducing that level of change (when a developer starts compiling against a newer version, they're in the position to easy make the needed change).  Just wanted to point it out to make sure others don't have a different opinion /cc @dnnsoftware/approvers

Reply:
Yeah, IMO I am ok with this, if a developer is rebuilding, then he would know about it right and could adjust quickl and easy.

Reply:
Depracation version has been resolved in latest commit, are we ok with the source breaking change?

Reply:
I'm okay with source breaking change